documentation/modules/operators/ref-operator-cluster.adoc

```diff
@@ -175,30 +175,33 @@ The timeout for building new Kafka Connect images with additional connectots, in
 This value should be increased when using Strimzi to build container images containing many connectors or using a slow container registry.
 
 `STRIMZI_FEATURE_GATES`:: Optional.
-Disables or enables feature gates to activate new features or change the operator behavior.
-For more details about the feature gates, see xref:ref-operator-cluster-feature-gates-{context}[].
+Enables or disables feature gates that activate new features or change the behavior of the operator.
+For more information about each feature gate, see xref:ref-operator-cluster-feature-gates-{context}[].
 
 [id='ref-operator-cluster-feature-gates-{context}']
 == Feature gates
 
-Strimzi operators support feature gates to turn functionality on and off.
-When applied, feature gates alter the behavior of the operator.
-You can change the default state for each feature gate to enable or disable them.
+Strimzi operators support _feature gates_ to enable or disable certain features and functionality. 
+When enabled, a feature gate alters the behavior of the relevant operator. 
```

Comment:
I'm not clear on the difference between introducing a feature and changing the behavior of the operator.  Is it that the feature gate changes the behavior of the operator by introducing new functionality?

Reply:
Yes, that's how I understand it. I updated this to:

> Enabling a feature gate changes the behavior of the relevant operator and introduces the feature to your Strimzi deployment.